src/OrbitGl/App.cpp

```diff
@@ -1889,6 +1889,11 @@ orbit_base::Future<ErrorMessageOr<void>> OrbitApp::LoadSymbols(
     return it->second;
   }
 
+  const ModuleData* module_data = GetModuleByPathAndBuildId(module_file_path, module_build_id);
+  if (module_data->is_loaded()) {
+    return ErrorMessageOr<void>{outcome::success()};
```

Comment:
I would add an `ERROR` though, and a comment pointing to the bug. At first glance this shouldn't be possible.

Reply:
Yep. Will do.

Reply:
Done.